client/public/solar-system.svg

```diff
@@ -0,0 +1 @@
.777 0 1.137-3.174 2.363-5.621 3.917-7.607-.008-.021-.023-.035-.031-.056z" fill="#f3f0f3"/></g><g><path d="m386.085 132.725c-8.054 2.885-11.475 6.306-14.36 14.36-.639 1.784-3.138 1.784-3.777 0-2.885-8.054-6.306-11.475-14.36-14.36-1.784-.639-1.784-3.138 0-3.777 8.054-2.885 11.475-6.306 14.36-14.36.639-1.784 3.138-1.784 3.777 0 2.885 8.054 6.306 11.475 14.36 14.36 1.785.639 1.785 3.138 0 3.777z" fill="#fff"/></g><g><g><path d="m360.838 125.802c-.15-.054-.27-.137-.395-.215-1.847 1.296-4.069 2.364-6.856 3.362-1.784.639-1.784 3.138 0 3.777 8.054 2.885 11.475 6.306 14.36 14.36.639 1.784 3.138 1.784 3.777 0 1.021-2.85 2.114-5.112 3.451-6.983-2.879-8.01-6.303-11.424-14.337-14.301z" fill="#f3f0f3"/></g></g><g><path d="m196.476 488.475c-13.048 4.302-17.858 9.113-22.161 22.161-.6 1.819-3.19 1.819-3.79 0-4.302-13.048-9.113-17.858-22.161-22.161-1.819-.6-1.819-3.19 0-3.79 13.048-4.302 17.859-9.113 22.161-22.161.6-1.819 3.19-1.819 3.79 0 4.302 13.048 9.113 17.858 22.161 22.161 1.819.6 1.819 3.19 0 3.79z" fill="#fea613"/></g><g><path d="m161.333 478.157c-3.192 2.693-7.302 4.66-12.969 6.528-1.819.6-1.819 3.19 0 3.79 13.048 4.302 17.859 9.113 22.161 22.161.6 1.819 3.19 1.819 3.79 0 1.868-5.667 3.835-9.777 6.528-12.969-3.919-10.93-8.58-15.591-19.51-19.51z" fill="#fe9901"/></g><g><path d="m394.339 380.649c-14.14 4.612-19.243 9.715-23.855 23.855-.604 1.851-3.206 1.851-3.81 0-4.612-14.14-9.715-19.243-23.855-23.855-1.851-.604-1.851-3.206 0-3.81 14.14-4.612 19.243-9.715 23.855-23.855.604-1.851 3.206-1.851 3.81 0 4.612 14.14 9.715 19.243 23.855 23.855 1.851.604 1.851 3.207 0 3.81z" fill="#fff"/></g><g><path d="m355.862 370.604c-3.293 2.504-7.455 4.413-13.044 6.236-1.851.604-1.851 3.206 0 3.81 14.14 4.612 19.243 9.715 23.855 23.855.604 1.851 3.206 1.851 3.81 0 1.787-5.48 3.656-9.592 6.088-12.855-4.169-11.844-9.057-16.855-20.709-21.046z" fill="#f3f0f3"/></g><g><circle cx="252" cy="288" fill="#fed402" r="14.863"/></g><g><path d="m146.521 287.364-6.157 6.157c-.524.524-1.375.524-1.899 0l-6.157-6.157c-.524-.524-.524-1.375 0-1.899l6.157-6.157c.524-.524 1.375-.524 1.899 0l6.157 6.157c.524.524.524 1.374 0 1.899z" fill="#fed402"/></g><g><path d="m210.607 343.449-6.157 6.157c-.524.524-1.375.524-1.899 0l-6.157-6.157c-.524-.524-.524-1.375 0-1.899l6.157-6.157c.524-.524 1.375-.524 1.899 0l6.157 6.157c.524.525.524 1.375 0 1.899z" fill="#fed402"/></g><g><path d="m405.607 155.449-6.157 6.157c-.524.524-1.375.524-1.899 0l-6.157-6.157c-.524-.524-.524-1.375 0-1.899l6.157-6.157c.524-.524 1.375-.524 1.899 0l6.157 6.157c.524.525.524 1.375 0 1.899z" fill="#fed402"/></g><g><path d="m340.607 112.449-6.157 6.157c-.524.524-1.375.524-1.899 0l-6.157-6.157c-.524-.524-.524-1.374 0-1.899l6.157-6.157c.524-.524 1.375-.524 1.899 0l6.157 6.157c.524.525.524 1.375 0 1.899z" fill="#fed402"/></g><g><path d="m225.607 371.449-6.157 6.157c-.524.524-1.375.524-1.899 0l-6.157-6.157c-.524-.524-.524-1.375 0-1.899l6.157-6.157c.524-.524 1.375-.524 1.899 0l6.157 6.157c.524.525.524 1.375 0 1.899z" fill="#fff"/></g></g></svg>
```

Comment:
There are some not used properties in your SVGs. These properties are created by design tools like Adobe XD.
For example, you can remove id and data-name.
Take a look at https://www.svgminify.com/

Similar to Hanttone/capstone-project#8 (comment)